installer/dev/install.cpp

```diff
@@ -289,6 +300,40 @@ namespace WindowsAppRuntimeInstaller
     }
     CATCH_RETURN()
 
+    // RestartPushNotificationsLRP is best effort and non-blocking to Installer functionality
+    void RestartPushNotificationsLRP()
+    {
+        WindowsAppRuntimeInstaller::InstallActivity::Context::Get().SetInstallStage(WindowsAppRuntimeInstaller::InstallActivity::InstallStage::RestartPushNotificationsLRP);
+
+        IID pushNotificationsIMPL_CLSID;
+        HRESULT hClsIdFromString = CLSIDFromString(PUSHNOTIFICATIONS_IMPL_CLSID_WSTRING, &pushNotificationsIMPL_CLSID);
+
+        if (SUCCEEDED(hClsIdFromString))
```

Comment:
Why the if check? Above call isn't expected to fail in any way right?

Reply:
Just following good coding practice.

Reply:
The CLSIDFromString call is never expected to fail. So being defensing and logging that seems overkill. I believe we should just THROW if that call fails because it is unexpected and should never happen.

Reply:
Fine. I will do that.